packages/itwinui-css/src/utils/mixins.scss

```diff
@@ -142,3 +142,27 @@
     @content;
   }
 }
+
+// Hide borders and hover/focus effects, eg. input
+@mixin iui-invisible-borders {
```

Comment:
this doesn't need to go in utils.

also it be handled all in one line using `all: unset`